File: ejercicios-js/ejercicio4.js
console.log("ejercicio 4:validar si un número es primo")
function esPrimo(n) {
    if (n < 2) return false;
    for (let i = 2; i <= Math.sqrt(n); i++) {
        if (n % i === 0) return false;
    }
    return true;
}

// Generar un número aleatorio entre 1 y 100
const numero = Math.floor(Math.random() * 100) + 1;
const resultado = esPrimo(numero) ? "es primo" : "no es primo";

console.log(`El número generado es ${numero} y ${resultado}.`);

File: ejercicios-py/ejercicio4.py
import random

def es_primo(n):
    if n < 2:
        return False
    for i in range(2, int(n**0.5)+1):
        if n % i == 0:
            return False
    return True

# Generar número aleatorio
numero = random.randint(1, 100)
print(f"Número generado: {numero}")

# Verificar si es primo
if es_primo(numero):
    print("Es un número primo.")
else:
    print("No es un número primo.")